From 5d1ff7656b2b1b8e50d9f30b180ff3e10b3adf3e Mon Sep 17 00:00:00 2001
From: Jeremy Kerr <jk@ozlabs.org>
Date: Mon, 6 Jul 2015 17:58:24 +0800
Subject: [PATCH 1/3] console/ast2400: Fix SIO address for SUART configuration

The SUART1 base address is at 60 hex, not 60 decimal.

Luckily, we've been setting it (from g_uartBase) to the default value
of 0xf8.

Change-Id: If6e6a095871bee5b55355590a28087ccc2a6bf62
Signed-off-by: Jeremy Kerr <jk@ozlabs.org>
Reviewed-on: http://gfw160.aus.stglabs.ibm.com:8080/gerrit/18941
Tested-by: Jenkins Server
Tested-by: Jenkins OP Build CI
Tested-by: Jenkins OP HW
Reviewed-by: Richard J. Knight <rjknight@us.ibm.com>
Reviewed-by: A. Patrick Williams III <iawillia@us.ibm.com>
---
 src/usr/console/ast2400.C | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/usr/console/ast2400.C b/src/usr/console/ast2400.C
index b2b63c3..c463dcd 100644
--- a/src/usr/console/ast2400.C
+++ b/src/usr/console/ast2400.C
@@ -242,7 +242,7 @@ namespace CONSOLE
                 l_errl = writeSIOReg( 0x60, (g_uartBase >> 8) & 0xFF );
                 if (l_errl) { break; }
 
-                l_errl = writeSIOReg( 61, (g_uartBase & 0xFF) );
+                l_errl = writeSIOReg( 0x61, (g_uartBase & 0xFF) );
                 if (l_errl) { break; }
 
                 // Set the SerIRQ
-- 
2.4.4

